.github/workflows/release.yml

```diff
@@ -25,6 +25,12 @@ jobs:
     # make test
     - name: Test
       run: go test -v ./...
+    # make test
+    - name: Test CLI module
+      run: go test -v ./cmd/grafana-app-sdk/...
+    # make test
+    - name: Test plugin module
+      run: go test -v ./plugin/...
```

Comment:
Hm, do you think we can update our workflows to rely on `make` instead? I'm not happy with the duplication we have here.

Reply:
IIRC there was some problem with using make in the workflow, but let me try and see what happens.

Reply:
Looks like it worked, nvm, good call